ios/engine/KMEI/KeymanEngine.xcodeproj/project.pbxproj

```diff
@@ -1035,7 +1022,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
-				ONLY_ACTIVE_ARCH = YES;
+				ONLY_ACTIVE_ARCH = NO;
```

Comment:
For debug builds, if they aren't getting published I think we can get away with `ONLY_ACTIVE_ARCH=YES` to avoid doing extra work unless we plan on releasing debug version as well.